src/DetailsView/Styles/detailsview.scss

```diff
@@ -417,7 +417,6 @@ div.insights-file-issue-details-dialog-container {
                 word-break: break-word;
             }
             width: 100%;
-            padding-bottom: 1px;
```

Comment:
Why is this removed?

Reply:
I added the padding to the details-view-body-content-container instead